extensionsGallery.json

```diff
@@ -5331,7 +5331,66 @@
 				],
 				"statistics": [],
 				"flags": ""
-			}
+			},
+
+			{
+					"extensionId": "106",
```

Comment:
Please also: 

1. Update the insiders gallery file
2. Update the ResultCount value at the end of each file (increase it by 1)

Reply:
making these updates now

Reply:
@Charles-Gagnon should now be updated